geometry_msgs/Point position
std_msgs/Int32 ymin
std_msgs/Int32 xmin
std_msgs/Int32 ymax
std_msgs/Int32 xmax
